lib/byron/src/Cardano/Wallet/Byron/Network.hs

```diff
@@ -171,7 +187,7 @@ data instance Cursor (m Byron) = Cursor
     (TQueue m (ChainSyncCmd m))
 
 -- | Create an instance of the network layer
-newNetworkLayer
+withNetworkLayer
```

Comment:
definitely better naming, sometimes I think it should be part of our coding style to enforce naming such things like that

Reply:
Yeah, it really is a recurring pattern for resource allocation.